docs/contribute.rst

```diff
@@ -84,6 +84,18 @@ Set your editor up to use isort as seen in the Makefile (format command) before
 
 You can also bulk apply formatting changes to all files by running ``make format``
 
+Labels in PRs and Issues
```

Comment:
This should not be here. It should be part of the flow here and made to match this text. 
![2023-03-01_21-13](https://user-images.githubusercontent.com/1658405/222255195-63b94e00-6047-48ce-b849-b46fd90ef1ce.png)